source/components/NPSCollect/index.js

```diff
@@ -15,23 +15,34 @@ class NPSCollect extends React.Component {
   constructor () {
     super()
 
-    this.state = { sending: false, score: -1 }
+    this.state = {
+      sending: false,
+      submittedScore: false,
+      submittedFeedback: false,
+      score: -1
+    }
 
     this.submitFeedback = this.submitFeedback.bind(this)
     this.submitScore = this.submitScore.bind(this)
+    this.submitError = this.submitError.bind(this)
     this.handleFeedbackChanged = this.handleFeedbackChanged.bind(this)
     this.handleScoreSelected = this.handleScoreSelected.bind(this)
   }
 
+  submitError (err) {
```

Comment:
`submissionError`?  I don't feel super strongly about it but `.catch(this.submitError)` made me think it was posting to sentry or something :shrug: